### PR TITLE
doc change for recent rate limits

### PR DESCRIPTION
**Pull request by haozhao-okta**
## Description:
- **What's changed?**  
  - /login/step-up/redirect is included to Dynamic Scale and Workforce Multiplier list
  - Increase default limit for /login/login.htm
  - /oauth2/v1/authorize and /oauth2/{authorizationServerId}/v1/authorize now have their own rate limit buckets
- **Is this PR related to a Monolith release?** 2021.09.0

### Resolves:

* [OKTA-423590](https://oktainc.atlassian.net/browse/OKTA-423590)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 1]
```diff
@@ -20,18 +20,20 @@ This page provides the API rate limits for authentication and end-user activitie
...
-| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/{authorizationServerId}/v1` except public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
-| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients` and public metadata endpoints (see Note below) | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/{authorizationServerId}/v1` except `/oauth2/{authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| `/oauth2/{authorizationServerId}/v1/authorize`                                                                         | 300              | 1200             | 1200    | 1200       | 2000
+| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients`, `/oauth2/v1/authorize` and public metadata endpoints (see Note below) | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```
[the remark this answers is not in the document]
Comment (haozhao-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:29]: `/oauth2/v1/clients` is doced on [another page](https://github.com/okta/okta-developer-docs/blob/40dd1c3b30052c97233f1918e68da530113771d5/packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md)

**Review by christinechoy-okta — APPROVED** [review]
Just a couple of style nits, otherwise LGTM!

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 2]
```diff
@@ -20,18 +20,20 @@ This page provides the API rate limits for authentication and end-user activitie
...
 | **Verify a factor:**<br>`/api/v1/authn/factors/{factorIdOrFactorType}/verify` only                                     | 100              | 600              | 600     | 600        | 500                   |
 | **Get session information:**<br>`/api/v1/sessions`                                                                     | 100              | 600              | 600     | 600        | 750                   |
-| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/{authorizationServerId}/v1` except public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
-| **OAuth2 requests for the Org Authorization Server:**<br>`/oauth2/v1` except `/oauth2/v1/clients` and public metadata endpoints (see Note below) | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
+| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/{authorizationServerId}/v1` except `/oauth2/{authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 3]
```diff
@@ -20,18 +20,20 @@ This page provides the API rate limits for authentication and end-user activitie
...
+| `/oauth2/{authorizationServerId}/v1/authorize`                                                                         | 300              | 1200             | 1200    | 1200       | 2000
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:28]:
```suggestion
| `/oauth2/${authorizationServerId}/v1/authorize`                                                                         | 300              | 1200             | 1200    | 1200       | 2000
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md (hunk 2 above)
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:27]:
```suggestion
| **OAuth2 requests for Custom Authorization Servers:**<br>`/oauth2/${authorizationServerId}/v1` except `/oauth2/${authorizationServerId}/v1/authorize` and public metadata endpoints (see Note below)  | 300 | 1,200 | 1,200     | 1,200       | 2,000                  |
```

**Review by IanHakes-okta — APPROVED** [review]
Conflicts need to be resolved but otherwise it's good to go

**Pull request merged**